client/src/components/organisms/Banner.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FunctionComponent } from "react";
+import BannerCarousel from "../atoms/BannerCarousel";
+import DonorHomepageConfig from "../../config/donorHomepageConfig.json";
+
+const Banner: FunctionComponent = () => {
```

Comment:
nit: call the component `DonorHomepageBanner` or `HomepageBanner` so no one thinks it's a generic banner component